chrome-extension/popup.css

```diff
@@ -0,0 +1,114 @@
+body {
+  font-family: 'Roboto';
+  font-size: 1rem;
+  margin: 0;
+  position: relative;
+  width: 325px;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+nav {
+  align-items: center;
+  background-color: #004aad;
+  color: white;
+  font-size: 1.75rem;
```

Comment:
Most of these properties seem sorted so I assume this one should be as well.

Reply:
Oh I missed this one, thanks I'll change that!